scripts/content-repo-preview/build.ts

```diff
@@ -41,35 +39,19 @@ async function main() {
   }
 
   // copy public files
-  console.log('📝 copying files in the public folder')
+  console.log('📝 Copying files from "./public" to "../"')
   execFileSync('cp', ['-R', './public', '../'])
 
-  /**
-   * Remove dirs in `src/pages` which are not associated with the product
-   */
-  const pagesDir = path.join(cwd, 'pages')
-
-  const pagesDirs = (
-    await fs.promises.readdir(pagesDir, { withFileTypes: true })
-  ).filter((ent) => ent.isDirectory())
-
-  for (const dir of pagesDirs) {
-    if (!dir.name.includes(repo) && dir.name !== 'home') {
-      console.log(`🧹 removing pages for ${dir.name}`)
-      await fs.promises.rm(path.join(pagesDir, dir.name), {
-        recursive: true,
-      })
-    }
-  }
```

Comment:
removing this prior build-time optimization so that...
- we don't need to support additional complexity, in code, for different consumers
- we don't introduce confusing 404 pages on deploy previews